ecl/eclcmd/eclcmd_core.cpp

```diff
@@ -886,13 +886,31 @@ class EclCmdRun : public EclCmdWithEclTarget
 
         switch (state)
         {
-        case WUStateRunning:
-            fprintf(stderr, "Timed out waiting for %s to complete, workunit is still running.\n", wuid.str()); //server side waiting timed out
-            break;
         case WUStateCompleted:
             break;
+        case WUStateCompiled:
```

Comment:
Still not sure about this one case. If I request that a workunit be compiled, and it ends up in state compiled, it's not an error...